.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
VS code settings are usually not committed, you can add this to your gitignore file.